src/assets/css/twitterwall.css

```diff
@@ -42,9 +43,11 @@
 	
 }
 .container-tweet{
+	box-sizing: border-box;
 	width:504px;
 	height:240px;
-	background-color:rgba(0, 0, 0, 0.59);;
+	background-color:rgba(0, 0, 0); /* For IE8 and IE9 */
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fallback background-color (hex or RGB) should precede RGBA background-color.](https://www.codacy.com/app/triplez/open-event-webapp/file/2416367476/issues/source?bid=3233857&fileBranchId=3235518#l50)